docker-compose.yml

```diff
@@ -1,15 +1,16 @@
 ---
-version: '3.9'
```

Comment:
Yes I see what you mean, the spec lists `version` as deprecated, and it seems it's no longer needed with recent versions. Might Do we just want to take this out completely?